src/gps/read_usb_gps.cpp

```diff
@@ -51,6 +51,7 @@ point_t gpsToMeters(double lon, double lat) {
 transform_t readGPS() {
   gps_mutex.lock();
   transform_t tf = most_recent_tf;
+  fresh_data = false;
```

Comment:
Oh hmmm. I think I fixed this in the branch Abhay has been using but it didn't make it back to this branch. Sorry.